GNUmakefile

```diff
@@ -37,11 +37,10 @@ attach:
 	dlv --listen=:2345 --headless=true --api-version=2 --accept-multiclient attach $$(pgrep terraform-provider-artifactory)
 
 acceptance: fmtcheck
-	export TF_ACC=1
-	test -n ARTIFACTORY_USERNAME && test -n ARTIFACTORY_PASSWORD && test -n ARTIFACTORY_URL \
+	export TF_ACC=true && \
```

Comment:
This doesn't seem to work at all. I think you can just delete it

Reply:
Which part doesn't work?

Reply:
the `export` part - You need to export before running the tests

Reply:
The `&&` part does the trick. Works for me in my shell.